docs/reference-guides/interactivity-api/iapi-faq.md

```diff
@@ -0,0 +1,129 @@
+# Frequently Asked Questions
+
+## How does the Interactivity API work under the hood?
+
+Its three main components are:
+
+- [Preact](https://preactjs.com/) combined with [Preact Signals](https://preactjs.com/guide/v10/signals/) for hydration, client logic, and client-side navigation.
+- HTML Directives that can be understood by both the client and server.
+- Server-side logic, handled by the [HTML_Tag_Processor](https://make.wordpress.org/core/2023/03/07/introducing-the-html-api-in-wordpress-6-2/).
+
+## Why Preact to build the directives system? Why not React or another JavaScript framework?
+
+Preact has a number of advantages over React and other JavaScript frameworks like Vue, Svelte, or Solid in the context of the frontend (which is the focus of the Interactivity API):
+
+- It’s small: 8kB, including [hooks](https://preactjs.com/guide/v10/hooks/) and [signals](https://preactjs.com/blog/introducing-signals/).
+- It’s battle-tested.
+- It’s performant (even more when used with signals).
+- It’s compatible with React (through preact/compat, useful to share the same Editor components for some use cases where SSR is not important, and the components are very complex, like an e-commerce cart and checkout, for example).
+- It’s HTML-friendly (unlike React).
```

Comment:
I'm not sure what this means, maybe this could link to context or be removed.

Reply:
I agree on removing these ones.
 - We don't support components yet.
 - We should not talk about performance without metrics.
 - It's battle tested is not so important, as being in Core means already that.

Reply:
Thanks for the feedback.
I have removed these points 👍